### DISCUSSION TITLE
quick question: anydesk ?

### DISCUSSION THREAD

**Discussion by sgoudelis** [jekhokie/raspberry-noaa-v2#865] [Q&A]
Good morning,

I am interested in finding out why this project requires Anydesk installed and running ? It seems superfluous and somewhat suspicious. Is there any clarification about this somewhere ? I can't find anything.

**Comment by sgoudelis** [comment]
Also I can't get the install/update script to work after doing a git pull because of the anydesk package not installing properly.

**Reply by sr55** [reply]
You can remove the step from ansible/roles/common/tasks/dependencies.yml

I also note there are steps to open various ports that don't need to be open.  I wonder if it's something else the author has installed. Either way, it shouldn't need RDP, VNC etc either.

**Comment by Itsisme** [comment]
If you want to run anydesk you need also to add a softlink...

systemctl status anydesk.service

By default I got :

/usr/bin/anydesk: error while loading shared libraries: libbrcmGLESv2.so: cannot open shared object file: No such file or directory

ln -s /usr/lib/arm-linux-gnueabihf/libGLESv2.so /usr/lib/libbrcmGLESv2.so

ldconfig and after that it starts ok....

systemctl enable anydesk.service
systemctl start anydesk.service

Chrs..
